stix_shifter_modules/bigfix/configuration/config.json

```diff
@@ -1,7 +1,8 @@
 {
     "connection": {
         "type": {
-            "default": "BigFix"
+            "default": "bigfix",
+            "displayName": "IBM® BigFix"
```

Comment:
`"displayName": "BigFix"`

its not an IBM product anymore. so we need to remove all mentions of IBM with BigFIx